src/html.pre.js

```diff
@@ -55,9 +56,29 @@ function pre(context) {
   }
 
   // ensure content.data is present
-  if (!context.content.data) {
-    context.content.data = {};
-  }
+  content.data = content.data || {};
+
+  // extract metadata
+  const { meta = {} } = content;
+  // description: text from paragraphs with 10 or more words
+  const desc = $sections
+    .filter('.default')
+    .find('p')
+    .filter(function minLength() {
+      return $(this).text().split(' ').length >= 10;
+    })
+    .map(function padWithSpaces(i) {
+      if (i > 0) $(this).text(` ${$(this).text()}`);
+      return this;
+    })
+    .text()
+    .trim();
+  // truncate after 25 words
+  const words = desc.split(' ');
```

Comment:
if you anyways need the words, why not return the words in your `map` function directly?

Reply:
Good point!